hack/make/doc/api-ref.mk

```diff
@@ -0,0 +1,3 @@
+doc/api-ref: manifests
+	python3 ./hack/doc/custom_resource_params_to_md.py ./config/crd/bases/dynatrace.com_dynakubes.yaml > ./doc/dynakube-api-ref.md
+	python3 ./hack/doc/custom_resource_params_to_md.py ./config/crd/bases/dynatrace.com_edgeconnects.yaml > ./doc/edgeconnect-api-ref.md
```

Comment:
would it make sense to start to have folders in the `doc` folder 🤔 

these could go into `/doc/api/...` for example

Reply:
yeah good idea, will do it.